test/test_xla_sharding.py

```diff
@@ -32,12 +30,30 @@ def test_xla_sharded_tensor(self):
     assert t3.size() == t1.size(
     ), "Sharded output should return unpartitioned tensor size."
 
-    device_ids = np.array(range(num_devices))
-    device_assignment = list(device_ids.reshape(mesh_shape))
-    # TODO(yeounoh) sharding_spec is broken. The intention is to
-    # check if the returned sharding spec holds the correct device
-    # assignment as intended.
-    # assert device_assignment == t1_sharded.sharding_spec[0]
+    # TODO(yeounoh) Check if the returned sharding spec holds the correct device
+    # assignments.
+
+  @unittest.skipIf(
+      len(xm.get_xla_supported_devices("GPU")) > 0,
+      "SPMD is currently not supported on GPU.")
```

Comment:
I think it is supported on GPU, but we don't need to support it in first version.

Reply:
+1 rephrasing the comment to avoid confusion. I really think we should focus on the TPU (and CPU) first. Once we hit all the milestones, adding GPU support should be more manageable/easier.

Reply:
Done